src/bootstrap/builder.rs

```diff
@@ -458,6 +458,12 @@ impl<'a> Builder<'a> {
             stage = compiler.stage;
         }
 
+        let cgus = if mode == Mode::Libstd {
+            self.config.rust_codegen_units
+        } else {
+            self.config.rustc_codegen_units.unwrap_or(self.config.rust_codegen_units)
```

Comment:
Could we make `rustc_codegen_units` do what it says on the tin and *just* apply to rustc (rather than it being an 'all-but-std' flag)? That seems to be where the majority of time goes anyway when I build the compiler.

Reply:
Sure yeah. This I expect to be one of those "obscure options that no one ever touches", but I can change it after this PR lands or if it bounces.